### ISSUE TITLE
Support for Pip 7.0

### ISSUE THREAD

**Issue by mythmon** [erikrose/peep#95]
Pip 7.0 deprecated the `--no-use-flag`, and replaced it with another option (`--no-binary=:all:`, which may also be used on only certain packages). The old option still works, but is deprecated and is internall translated into the `--no-binary` case. Additionally, it seems that the internal handling of the option changed, so use_wheel is not a valid argument to PackageFinder anymore.

Traceback:

```
peep: (2, 4, 0)
python: '2.7.9 (default, Dec 11 2014, 04:42:00) \n[GCC 4.9.2]'
pip: '7.0.1'
Command line:  ['./scripts/peep.py', 'install', '-r', 'requirements/default.txt', '-r', 'requirements/dev.txt', '--no-binary=:all:']
Traceback (most recent call last):
  File "./scripts/peep.py", line 902, in <module>
    exit(main())
  File "./scripts/peep.py", line 874, in main
    return commands[argv[1]](argv[2:])
  File "./scripts/peep.py", line 834, in peep_install
    for path in req_paths))
  File "./scripts/peep.py", line 834, in <genexpr>
    for path in req_paths))
  File "./scripts/peep.py", line 795, in downloaded_reqs_from_path
    finder = package_finder(argv)
  File "./scripts/peep.py", line 335, in package_finder
    return PackageFinder(index_urls=index_urls, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'use_wheel'
```

**Comment by mythmon** [comment]
Oh, I see this is already being fixed in #94. Excellent, I'll close this out.

**Comment by kouk** [comment]
Well it hasn't been merged yet.
